### PR TITLE
release: stable-2.0.0

### PR DESCRIPTION
**Pull request by joejulian**
Release of stable-2.0.0

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,9 +1,45 @@
...
+* \[prometheus-operator\] Upgrade to version [0.38.1](https://github.com/coreos/prometheus-operator/releases/tag/v0.38.1)
+    - \[prometheus\] Upgrade to version [2.17.2](https://github.com/prometheus/prometheus/releases/tag/v2.17.2)
+    - \[grafana\] Upgrade to version [6.7.3](https://github.com/grafana/grafana/releases/tag/v6.7.3) ([#281](https://github.com/mesosphere/kubernetes-base-addons/pull/281), [@branden](https://github.com/branden))
+* \[traefik\] fix an issue where `clusterhostname` can now be an ipaddress as well ([#286](https://github.com/mesosphere/kubernetes-base-addons/pull/286), [@GoelDeepak](https://github.com/GoelDeepak))
+* [dex-k8s-authenticator] Fix bug in init container that could remove custom CA certificate from main cluster login instructions ([#291](https://github.com/mesosphere/kubernetes-base-addons/pull/291), [@mhrabovcin](https://github.com/mhrabovcin))
+* \[traefik\] Distribute pods across nodes and zones when possible.
```
Comment (samvantran) [inline RELEASE_NOTES.md:13]: Nice, a lot of improvements in this release!

Can we group all the notes for the same addon together? Seeing a lot of traefik and prometheus notes scattered throughout.

**Comment by samvantran** [comment]
Doubt it's related since these are just release notes but any cause for alarm from the failed `dispatch-integration-test`?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (joejulian) [reply RELEASE_NOTES.md:13]: I'd prefer not to in order to keep the notes attached to their individual PRs and contributors.

Comment (joejulian) [reply RELEASE_NOTES.md:13]: > a lot of improvements in this release

Unfortunately, that's because it's been 6 weeks since the last stable release.

**Comment by joejulian** [comment]
> Doubt it's related since these are just release notes but any cause for alarm from the failed dispatch-integration-test?

OOM killed. See #347

**Comment by joejulian** [comment]
D'oh! Mouse overshot.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (samvantran) [reply RELEASE_NOTES.md:13]: you'd still be able to maintain that info. it's just bringing them under the same grouping so it's easier to see all the updates for that addon e.g.
```
[traefik]
- fix an issue where `clusterhostname` can now be an ipaddress as well ([#286](https://github.com/mesosphere/kubernetes-base-addons/pull/286), [@GoelDeepak](https://github.com/GoelDeepak))
- access log is enabled by default ([#305](https://github.com/mesosphere/kubernetes-base-addons/pull/305), [@mhrabovcin](https://github.com/mhrabovcin))
- ...
```

Comment (joejulian) [reply RELEASE_NOTES.md:13]: Let's get the Makefile that does this merged first so the process is all in code, then work on automating that in future iterations?

**Pull request merged**